.ci/azure-pipelines-api-client.yml

```diff
@@ -0,0 +1,62 @@
+﻿parameters:
+  - name: LinuxImage
+    type: string
+    default: "ubuntu-latest"
+  - name: GeneratorVersion
+    type: string
+    default: "4.3.1"
+
+jobs:
+- job: GenerateApiClients
+  displayName: 'Generate Api Clients'
+  dependsOn: Test
+
+  pool:
+    vmImage: "${{ parameters.LinuxImage }}"
+
+  steps:
+    - task: DownloadPipelineArtifact@2
+      displayName: 'Download OpenAPI Spec Artifact'
+      inputs:
+        source: 'current'
+        artifact: "OpenAPI Spec"
+        path: "$(System.ArtifactsDirectory)/openapispec"
```

Comment:
You use `$(System.ArtifactsDirectory)/openapispec` here but `$(System.ArtifactsDirectory)/openapi` in the actual bash scripts.